client/lib/feed-stream-store/test/index.js

```diff
@@ -1,3 +1,11 @@
+jest.mock( 'lib/analytics', () => ( {} ) );
+jest.mock( 'lib/data-poller', () => require( './lib/data-poller' ) );
+jest.mock( 'lib/post-normalizer', () => require( './lib/post-normalizer' ) );
+jest.mock( 'lib/wp', () => require( './lib/wp' ) );
+jest.mock( 'reader/stats', () => ( {
```

Comment:
@bluefuton Any chance we Could port those to use the redux [analytics middleware](https://github.com/Automattic/wp-calypso/tree/master/client/state/analytics) instead?